symphony/Symphony/Plugins/GetWindowsPlugin.cs

```diff
@@ -41,6 +41,10 @@ public void cefGetScreenMedia(JObject arguments, JavaScriptPluginCallback callba
                     values.TryGetValue("sources", out sources);
                 }
 
+                if (sources == null) {
+                    sources = { "screen", "window" };

```

Comment:
we should do this instead:
if (sources == null || sources.Length == 0) 
  sources = new string[] { "screen", "window" };